### ISSUE TITLE
Strip version disclosures from response headers

### ISSUE THREAD

**Issue by jfrank-nih** [NCIOCPL/wcms-cde#45]

**Comment by jfrank-nih** [comment]
@blairlearn, we could bug the hosting team, or... everything passes through Akamai from the origin, correct? In which case we could strip out headers there.

**Comment by blairlearn** [comment]
No, this isn't a problem in CGDP, it shows up there because of how things are mapped through Akamai.  The correct fix is to address it in dynamic services. (Which is where I'll be moving this ticket momentarily.)
